tests/unittests/fcv_tests.py

```
@@ -232,11 +266,98 @@ def test_update_fc_version_pymongo_failure(
+                        # Mock the command responses
+                        mock_admin.command.return_value = {
+                            "featureCompatibilityVersion": {
+                                "version": str(fc_version)
+                            }
+                        }
+
+                        # Call the function to test
+                        _update_fc_version(False, mock_client)
+
+                        # Should issue a get to the fcv, but not a subsequent update
+                        mock_admin.command.call_count == 1
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)